makefile

```
@@ -69,6 +69,11 @@ psalm:
 phpunit:
 	vendor/phpunit/phpunit/phpunit
 
+#for only php8.0
+.PHONY: phpunitforphp8.0
+phpunitforphp8.0:
+	vendor/phpunit/phpunit/phpunit --exclude-group ignoreFor8.0
```

maybe make the naming more explicit
```suggestion
	vendor/phpunit/phpunit/phpunit --exclude-group ignoreWithPHP8.0
```